tools/smina/smina.xml

```diff
@@ -0,0 +1,364 @@
+<tool id="smina" name="smina" version="1.0">
+    <description>Scoring and Minimization with AutoDock Vina</description>
+    <requirements>
+        <requirement type="package" version="1.1.2">smina</requirement>
+    </requirements>
+    <stdio>
+        <exit_code range="1:" />
+    </stdio>
+    <command>
+        <![CDATA[ 
+        ln -s $input.receptor receptor.pdbqt;
+        ln -s $input.ligand ligand.pdbqt;
+        smina.static 
+          --receptor receptor.pdbqt 
+          --ligand ligand.pdbqt 
+
+          #if '$input.flex'
+              --flex $input.flex
+          #end if
+          #if '$input.flexres'
+              --flexres $input.flexres
+          #end if
+          #if '$input.flexdist_ligand'
+              --flexdist_ligand $input.flexdist_ligand
+          #end if
+          #if '$input.flexdist'
+              --flexdist $input.flexdist
+          #end if
+
+          --center_x $search_space.center_x 
+          --center_y $search_space.center_y
+          --center_z $search_space.center_z 
+          --size_x $search_space.size_x
+          --size_y $search_space.size_y
+          --size_z $search_space.size_z
+
+          #if '$search_space.autobox_ligand'
+              --autobox_ligand $search_space.autobox_ligand 
+          #end if 
+          #if '$search_space.autobox_add' 
+              --autobox_add $search_space.autobox_add
+          #end if
+          #if '$search_space.no_lig'=="true"
+              --no_lig
+          #end if 
+
+          --out ligand_out.pdbqt
+          #if '$output_sect.out_flex'=="true"
+              --out_flex output_flex
+          #end if                                                    
+          #if '$output_sect.log'=="true"
+              --log output_log
+          #end if
+          #if $output_sect.atom_terms=="true"
+              --atom_terms output_atom_terms
+          #end if
+          #if $output_sect.atom_term_data=="true"
+              --atom_term_data
+          #end if
+
+          --scoring $scoring_and_minimization.scoring 
+          #if $scoring_and_minimization.custom_scoring
+              --custom_scoring $scoring_and_minimization.custom_scoring
+          #end if
+          #if $scoring_and_minimization.custom_atoms
+              --custom_atoms $scoring_and_minimization.custom_atoms 
+          #end if
+          #if $scoring_and_minimization.score_only=="true"
+              --score_only 
+          #end if
+          #if $scoring_and_minimization.local_only=="true"
+              --local_only 
+          #end if
```

Comment:
If you change the param to boolean below, you can replace the whole `#if` block with just:
```
          $scoring_and_minimization.local_only
```